docs/book/src/reference/versions.md

```diff
@@ -72,14 +72,14 @@ These diagrams show the relationships between components in a Cluster API releas
 
 #### Core Provider (`cluster-api-controller`)
 
-|                   | v1.2 (v1beta1) (EOL) | v1.3 (v1beta1)    | v1.4 (v1beta1)    | v1.5 (v1beta1)    |
+|                   | v1.3 (v1beta1) (EOL) | v1.4 (v1beta1)    | v1.5 (v1beta1)    | v1.6 (v1beta1)    |
 |-------------------|----------------------|-------------------|-------------------|-------------------|
-| Kubernetes v1.18  | ✓ (only workload)    | ✓ (only workload) |                   |                   |
-| Kubernetes v1.19  | ✓ (only workload)    | ✓ (only workload) |                   |                   |
-| Kubernetes v1.20  | ✓                    | ✓                 |                   |                   |
-| Kubernetes v1.21  | ✓                    | ✓                 | ✓ (only workload) |                   |
-| Kubernetes v1.22  | ✓                    | ✓                 | ✓ (only workload) | ✓ (only workload) |
-| Kubernetes v1.23* | ✓                    | ✓                 | ✓                 | ✓ (only workload) |
+| Kubernetes v1.18  | ✓ (only workload)    |                   |                   |                   |
+| Kubernetes v1.19  | ✓ (only workload)    |                   |                   |                   |
+| Kubernetes v1.20  | ✓                    |                   |                   |                   |
+| Kubernetes v1.21  | ✓                    | ✓ (only workload) |                   |                   |
+| Kubernetes v1.22  | ✓                    | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) |
+| Kubernetes v1.23* | ✓                    | ✓                 | ✓ (only workload) | ✓ (only workload) |
```

Comment:
Looks like we dropped support for 1.27 in v1.4 with this change

I think this table should be correct:
```
|                   | v1.3 (v1beta1) (EOL) | v1.4 (v1beta1)    | v1.5 (v1beta1)    | v1.6 (v1beta1)    |
|-------------------|----------------------|-------------------|-------------------|-------------------|
| Kubernetes v1.18  | ✓ (only workload)    |                   |                   |                   |
| Kubernetes v1.19  | ✓ (only workload)    |                   |                   |                   |
| Kubernetes v1.20  | ✓                    |                   |                   |                   |
| Kubernetes v1.21  | ✓                    | ✓ (only workload) |                   |                   |
| Kubernetes v1.22  | ✓                    | ✓ (only workload) | ✓ (only workload) | ✓ (only workload) |
| Kubernetes v1.23* | ✓                    | ✓                 | ✓ (only workload) | ✓ (only workload) |
| Kubernetes v1.24  | ✓                    | ✓                 | ✓                 | ✓                 |
| Kubernetes v1.25  | ✓                    | ✓                 | ✓                 | ✓                 |
| Kubernetes v1.26  | ✓                    | ✓                 | ✓                 | ✓                 |
| Kubernetes v1.27  |                      | ✓                 | ✓                 | ✓                 |
```